support-frontend/assets/helpers/forms/paymentIntegrations/readerRevenueApis.ts

```diff
@@ -135,11 +135,11 @@ export type RegularPaymentFields =
 	| RegularAmazonPayPaymentFields;
 export type RegularPaymentRequestAddress = {
 	country: IsoCountry;
-	state: UsState | CaState | null;
-	lineOne: Option<string>;
-	lineTwo: Option<string>;
-	postCode: Option<string>;
-	city: Option<string>;
+	state?: UsState | CaState | null;
+	lineOne?: string;
+	lineTwo?: string;
+	postCode?: string;
+	city?: string;
```

Comment:
[This is a more idiomatic TypeScript way of having optional properties](https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties).